scss/button.module.scss

```diff
@@ -16,4 +16,10 @@
   .btn-outline-bg-#{$name}:hover {
     background-color: rgba($color, 0.2);
   }
+
+  .btnOutlineInfoBgFix-#{$name} {
+    &:hover {
+      background-color: rgba($color, 0.2) !important;
```

Comment:
`!important` is the only way we could fix it?

Reply:
without the `!important`, i couldn't get it to override the default `--bs-btn-hover-bg` setting